src/Magnum/Platform/EmscriptenApplication.cpp

```diff
@@ -89,10 +105,18 @@ bool EmscriptenApplication::tryCreate(const Configuration& configuration, const
         (glConfiguration.flags() & GLConfiguration::Flag::FailIfMajorPerformanceCaveat) != GLConfiguration::Flags{};
     attrs.enableExtensionsByDefault =
         (glConfiguration.flags() & GLConfiguration::Flag::EnableExtensionsByDefault) != GLConfiguration::Flags{};
+#if FALSE /* "EMSCRIPTEN_VERSION > 1.38.12 */ && OFFSCREEN_FRAMEBUFFER
+    // TODO! Good thing my Emscripten version is from the bronze era
+    // We probably never want to use this anyway, except when we have a
+    // WindowlessEmscriptenApplication
```

Comment:
We have `WindowlessEglApplication`, all in-browser GL tests use it. No need to have headless rendering here :)